docs/changelog.rst

```diff
@@ -14,6 +14,12 @@ New Features
 ~~~~~~~~~~~~
 - Added support for :attr:`fortnite_api.CosmeticType.SHOES`.
 
+Miscellaneous
+~~~~~~~~~~~~~
+- Add safeguards against Fortnite changes or messed up values within the API.
+    - All enums now can handle unknown values. This means that if the API returns a value that is not in the enum, it will be stored as an attribute on the enum object.
```

Comment:
```suggestion
    - All enums now can handle unknown values via an internally defined "Enum-like" class. This means that if the API returns a value that is not in the enum, it will be stored as an attribute on the enum object.
```